pom.xml

```diff
@@ -469,6 +480,31 @@
           <useSystemClassLoader>false</useSystemClassLoader>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+        <version>2.0.0</version>
+        <executions>
+          <execution>
+            <configuration>
+              <verbose>false</verbose>
+              <thirdPartyFilename>THIRD-PARTY-Licenses.txt</thirdPartyFilename>
+              <useMissingFile>true</useMissingFile>
+              <missingFile>THIRD-PARTY-Missed-Licenses.txt</missingFile>
```

Comment:
Why is the default name `src/license/THIRD-PARTY.properties` being changed to `THIRD-PARTY-Missed-Licenses.txt`?

Reply:
I've added src/license/THIRD-PARTY.properties and filled in the missing licenses.
The plugin will alter this file only if there is a dependency change that changes the licenses used.